tasks/test-processing/package.json

```diff
@@ -9,6 +9,9 @@
     "url": "https://github.com/nasa/cumulus",
     "directory": "tasks/test-processing"
   },
+  "publishConfig": {
```

Comment:
This is not required since @cumulus/test-processing is already in npm.  Adding this part for consistency.